### ISSUE TITLE
UUID --number-sharing false

### ISSUE THREAD

**Issue by xyz-nobody** [AsamK/signal-cli#1466]
When trying to use

signal-cli -u +XXXX updateAccount --discoverable-by-number false --number-sharing false

It doesn't generate a UUID. 

The phone number is always broadcast.

On the other hand, people who use signal 7, when they receive their messages, no longer see the number, only their UUID.

However, the parameter is taken into account : --discoverable-by-number false
People can no longer write to us via the telephone number.

**Comment by xyz-nobody** [comment]
PS : Can we write via the username?

Currently I have the impression that only the phone number and the UUID are supported.

**Comment by AsamK** [comment]
What do you mean it doesn't generate a UUID? signal accounts now always have an ACI (uuid). People who previously had your phone number may still see it after changing --number-sharing option.

You can send to usernames (or username links) with the `-u` parameter, see https://github.com/AsamK/signal-cli/blob/master/man/signal-cli.1.adoc#send

**Comment by xyz-nobody** [comment]
Sorry, I've just realized my mistake.
I did a lot of tests and it's impossible to know when we receive or send a message if the number is broadcast.

You always see his number when you receive a message.


Envelope from: “test” f7a295d1-be04-4d2f-93be-906210b9aa16 (device: 1) to +XX
Timestamp: 1708556695067 (2024-02-21T23:04:55.067Z)
Server timestamps: received: 1708556695061 (2024-02-21T23:04:55.061Z) delivered: 1708556699021 (2024-02-21T23:04:59.021Z)
Message timestamp: 1708556695067 (2024-02-21T23:04:55.067Z)
Body: Test
With profile key

However, the person who receives the message does not have the number. 

I thought I could see it when sending or receiving a message, but it's not possible.

Sorry, everything works fine with signal-cli 13 :)
